x/gamm/README.md

```diff
@@ -171,14 +171,6 @@ The `x/gamm` module supports the following message types:
 
 [MsgExitPool](https://github.com/osmosis-labs/osmosis/blob/v7.1.0/proto/osmosis/gamm/v1beta1/tx.proto#L44-L57)
 
-### MsgSwapExactAmountIn
```

Comment:
We should merge this in: https://github.com/osmosis-labs/osmosis/pull/3504 first to isolate gamm refactor into its own PR.

We can either merge #3504 firtst and then rebase this. Another approach is to only merge just the CL module w/o gamm refactor. CL module is independent from that

Reply:
Im good to rebase this after 3504 is merged!

Reply:
merged, thanks for reviews